client/stylesheets/_toCat.scss

```diff
@@ -4418,7 +4422,7 @@ On click the images will be displayed at normal size to complete the effect
     height: 475px;
     margin-bottom: 20px;
     padding: 10px;
-    border: 2px solid #fff;
+    border: 1px solid #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces